src/main/python/systemml/converters.py

```diff
@@ -19,18 +19,21 @@
 #
 #-------------------------------------------------------------
 
-__all__ = [ 'getNumCols', 'convertToMatrixBlock', 'convert_caffemodel', 'convert_lmdb_to_jpeg', 'convertToNumPyArr', 'convertToPandasDF', 'SUPPORTED_TYPES' , 'convertToLabeledDF', 'convertImageToNumPyArr']
+__all__ = [ 'getNumCols', 'convertToMatrixBlock', 'convert_caffemodel', 'convert_lmdb_to_jpeg', 'convertToNumPyArr', 'convertToPandasDF', 'SUPPORTED_TYPES' , 'convertToLabeledDF', 'convertImageToNumPyArr', 'getModelMean']
 
 import numpy as np
 import pandas as pd
 import os
 import math
+
 from pyspark.context import SparkContext
 from scipy.sparse import coo_matrix, spmatrix, csr_matrix
 from .classloader import *
 
 SUPPORTED_TYPES = (np.ndarray, pd.DataFrame, spmatrix)
 
+MODEL_MEAN = {'VGG-19':[103.939, 116.779, 123.68]}
```

Comment:
Yes, technically we can go with DataSet name, year date version etc. 
By doing so intuition to pick mean value for associated model gets lost. It becomes harder to DS to figure out which mean to pick up.
These mean values are associated with model, unless someone going to change dataset for same model and keep model name exactly same (which I am not sure).

At this point I have changed to DATASET_MEAN (though I don't like it) and name of that dataset as VGG_ILSVRC_19_2014.